package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
why is this change? there is no new package being installed so I think no need for this one to be modified

Reply:
It just seems to be there po ma'am even if I don't stage it for commit during that time.

Reply:
Removed duplicate package-lock.json